cmake/modules/gluecodium/gluecodium/details/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```

Comment:
Why "all" now instead of "bin"?

Reply:
We download `all` in SDK to have relevant deprecation messages and documentation. I'd say it might be also applicable for Gluecodium. But for Gluecodium it's not so important may be.